contents/docs/tutorials/feature-flags.md

```diff
@@ -0,0 +1,206 @@
+---
+title: How to Safely Roll Out New Features
+sidebar: Docs
+showTitle: true
+---
+<br />
+
+<small class="note-block centered">_Estimated Reading Time: 8 minutes ☕☕_</small>
+
+<br />
+
+
+![](../../images/tutorials/banners/feature-flags.png)
+
+<br />
+
+Rolling out new features in your product can really be a pain.
+
+As a company that is growing fast and [consistently putting out new functionality](/blog/we-ship-whenever), we know this very well. 
+
+You might have great tests and robust CI/CD workflows in place, but there's always the chance that something will go wrong, or that your users may not appreciate the change.
+
+To make this process less painful and safer, you can use feature flags.
+
+Feature flags allow you to safely deploy and roll back new features. It means you can deploy features and then slowly roll them out to your users. If something has gone wrong, you can roll back new features without having to re-deploy your application.
+
+This tutorial will walk you through creating and using a feature flag with PostHog, cover some basic use cases, and answer some of the questions you may have when using them.
+
+<span class='centered'>...</span>
+
+_Prefer to watch a video? Check out the [Feature Flags section of our demo video](https://youtu.be/aUILrrrlu50?t=950) for a shorter overview of this functionality._
+
+## Creating Feature Flags
+
+### Step 1: Navigate to 'Experiments'
+
+To find the feature flags page, click on 'Experiments' on the left sidebar in PostHog.
+
+![Feature Flags Page](../../images/tutorials/feature-flags/feature-flags-page.png)
+
+### Step 2: Creating a New Flag
+
+While on the 'Experiments' page, click on the blue '+ New Feature Flag' button. This will open up a menu on the right side of the page, like so:
+
+![Create Feature Flag](../../images/tutorials/feature-flags/create-flag.png)
+
+### Step 3: Configuring Your Flag
+
+With the menu open, you will see there are a few options you can configure for your flag. Here's the purpose of each one:
+
+**Name**
+
+A memorable name for you and your team to refer to the flag. 
+
+**Key**
+
+Used to refer to your flag in your codebase. While the flag name does not affect the implementation, the key does. When adding the feature flag to your code, you will always use the key, never the name. In addition, changing a flag key may [affect the persistence of your flag](/docs/features/feature-flags#feature-flag-persistence).
+
+<div class='note-block'><b>Pro tip:</b> Make sure you choose a key for your flag that is descriptive and styled consistently with your codebase. Changing a flag key will require you to update your codebase and may harm persistence, so it's best to avoid doing so.</div><br />
+
+**Active**
+
+This toggle is an important part of feature flags. It means you can turn feature flags on only when you're ready to do so, or off if something goes wrong. It can be used as a "kill switch" for features.
+
+**Filter by user properties**
```

Comment:
Suggest: make more use-case driven ie **Roll out to specific users**

Reply:
This is based off of the app's exact wording

Reply:
ok, let's leave - makes sense!